samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -203,13 +226,56 @@ static struct bt_audio_capability_ops lc3_ops = {
 	.release = lc3_release,
 };
 
+
+#if defined(CONFIG_LIBLC3CODEC)
+
+static void stream_recv_lc3_codec(struct bt_audio_stream *stream, struct net_buf *buf)
+{
+	uint8_t err;
+	/* TODO: If there is a way to know if the controller supports indicating errors in the
+	 *       payload one could feed that into bad-frame-indicator. The HCI layer allows to
+	 *       include this information, but currently there is no controller support.
+	 *       Here it is assumed that reveiving a zero-length payload means a lost frame -
+	 *       but actually it could just as well indicate a pause in the stream.
+	 */
+	const uint8_t bad_frame_indicator = buf->len == 0 ? 1 : 0;
+	uint8_t *in_buf = (bad_frame_indicator ? NULL : buf->data);
```

Comment:
Hmm, apparently we discard the `const struct bt_iso_recv_info *info` in BAP, so you won't ever get that information directly. Maybe we'll need to fix that.

Reply:
Yes that would be ideal - also if controllers would actually issue ISO data packets over HCI when they detect a failure to receive :-) (I have only seen one controller that does that.)

Reply:
The fact that they can, should be enough reason that they should :) For something like audio, that information is of significant worth.